charts/unleash/templates/deployment.yaml

```diff
@@ -36,7 +36,7 @@ spec:
               valueFrom:
                 secretKeyRef:
                   name: {{ if .Values.dbConfig.useExistingSecret.name }}{{ .Values.dbConfig.useExistingSecret.name }}{{ else }}{{ .Values.postgresql.fullnameOverride }}{{ end }}
-                  key: {{ if .Values.dbConfig.useExistingSecret.key }}{{ .Values.dbConfig.useExistingSecret.key }}{{ else }}postgres-password{{ end }}
+                  key: {{ if .Values.dbConfig.useExistingSecret.key }}{{ .Values.dbConfig.useExistingSecret.key }}{{ else }}password{{ end }}
```

Comment:
@gastonfournier 
`postgres-password` is the Secret key where a password for `postgres` user is stored. https://github.com/bitnami/charts/blob/1601a1b831d4c45dbc285ed86c4b83ff52fbeb1f/bitnami/postgresql/templates/secrets.yaml#L17
The password for `unleash` user is stored in `password` key.